### PR TITLE
feat: added html-css

### PR DESCRIPTION
**Pull request by maxnoodle**

### CODE REVIEW

**Review by Aierbote — DISMISSED** [review]
Alot to process, but pretty good

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<html>
```
Comment (Rizaaal) [inline index.html]: aggiungere doctype?

**Review by Aierbote — APPROVED** [review]
Correct doctype 👍

**Review by Rizaaal — APPROVED** [review]
ok

**Pull request merged**